docs/roadmap.md

```diff
@@ -96,35 +96,34 @@ For questions or status updates on either features listed here, or features not
+| `<SwapChainPanel> `                                    | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Intellisense, Hot Reload, Live Visual Tree             | &#128311; | &#128311; | &#128994; (packaged apps only)  | &#128994;          | &#128994;     |
+| Chromium-based WebView2                                | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Title bar customization                                | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Fluent Shadows                                         | &#128994;                              | &#128994;                             | &#128994;                           | &#128994;
+| Input validation for data fields                       | &#128311; | &#128311; | &#128311;              | &#128311; | &#128994;     |
+| Supports multiple top-level windows on the same thread | &#128311; | &#128311; | &#128311;              | &#128994;          | &#128994;     |
+| Support multiple top-level windows on separate threads | &#128311; | &#128311; | &#128311;              | &#128311; | &#128994;     |
+| Drag and drop                                          | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| RenderTargetBitmap                                     | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Mouse cursor customization                             | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Animated Gif support                                   | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| VirtualSurfaceImageSource (VSIS) support               | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| In-app acrylic                                         | &#128994;          | &#128994;          | &#128994;                       | &#128994;          | &#128994;     |
+| Background acrylic                                     |            |            |                         | &#128994;          | &#128994;     |
+| Mica                                                   |            |            |                         | &#128994;          | &#128994;     |
+| XAML Islands                                           |            |            |                         | &#128311; | &#128994;     |
+| Media Controls (e.g. `<MediaPlayerElement>`)           |            |            |                         |            | &#128994;     |
```

Comment:
We shouldn't put an experimental/diamond here until Crider gets back and we have a discussion about it.  For 1.1, until we hear otherwise, it should be blank (not supported).

Reply:
Assuming you meant XAML Islands here.

Reply:
I meant there should not be an experimental diamond on Media Controls -- that hasn't been decided in the 1.1 timeframe.

Islands _should_ be experimental in 1.1 timeframe, so we should add that back.

Reply:
Mike also pointed out that Islands should be listed as experimental under 0.8, too.  Sorry, I didn't catch that earlier.